### PR TITLE
Don't use `lib` nuget package directory

### LINKED ISSUE #45: Packaging.Target should not have msbuild tasks in `lib` directory
`lib` should be used for libraries meant to be consumed by application. Place those files to `build` alongside with `.targets`

### PR DESCRIPTION
**Pull request by kekekeks**
Fixes #45

### CODE REVIEW

File: Packaging.Targets/Packaging.Targets.csproj [hunk 1]
```diff
@@ -22,6 +22,12 @@
   <PropertyGroup Condition="'$(Configuration)|$(TargetFramework)|$(Platform)'=='Debug|netstandard1.5|AnyCPU'">
     <PlatformTarget>AnyCPU</PlatformTarget>
   </PropertyGroup>
+  <!-- Copy build task assemby to build directory -->
+  <PropertyGroup>
+    <BuildOutputTargetFolder>tools</BuildOutputTargetFolder>
```
Comment (qmfrederik) [inline Packaging.Targets/Packaging.Targets.csproj]: Did you plan to use this variable in the `PackagePath` values?

File: Packaging.Targets/Packaging.Targets.csproj [hunk 2]
```diff
@@ -22,6 +22,12 @@
...
+    <!-- Suppress warning about lib being empty -->
+    <NoPackageAnalysis>true</NoPackageAnalysis>
```
Comment (qmfrederik) [inline Packaging.Targets/Packaging.Targets.csproj]: Can we suppress that individual warning instead? Will users who add a reference to Packaging.Targets get a warning?

File: Packaging.Targets/Packaging.Targets.csproj (hunk 1 above)
Comment (kekekeks) [reply Packaging.Targets/Packaging.Targets.csproj]: It's a standard property that's needed for Packaging.Targets.dll to be placed into `tools` directory.

File: Packaging.Targets/Packaging.Targets.csproj (hunk 2 above)
Comment (kekekeks) [reply Packaging.Targets/Packaging.Targets.csproj]: No idea, quick googling gave me no results.
No, there won't be any warnings for people who just reference the package.

File: Packaging.Targets/Packaging.Targets.csproj (hunk 1 above)
Comment (qmfrederik) [reply Packaging.Targets/Packaging.Targets.csproj]: makes sense, thanks.

File: Packaging.Targets/Packaging.Targets.csproj (hunk 2 above)
Comment (qmfrederik) [reply Packaging.Targets/Packaging.Targets.csproj]: OK, if this doesn't impact the end users, it's fine with me 😄

**Pull request merged**